src/redux/initialState.js

```diff
@@ -41,7 +41,7 @@ const initialState = {
       promo: 'sale',
       newFurniture: true,
       favourite: false,
-      addToCompare: true,
+      addToCompare: false,
```

Comment:
`addToCompare` brzmi jak funkcja, lepiej `addedToCompare` lub `isAddToCompare` ewentualnie jeszcze jakoś inaczej co sugeruje że to pole mówi czy produkt został dodany do porównania czy nie